### ISSUE TITLE
Qwen3.5-FP8 Output Mangled on Blackwell

### ISSUE THREAD

**Issue by sempervictus** [guoqingbao/vllm.rs#250]
`Qwen3.5-122B-A10B-FP8` without any quantization on 4xSM120 (`flashinfer,cutlass,graph,cuda,nccl,python`) produces:
```shell
$ aichat please identify your model capabilities and training cutoff date
0-�esTR#‐sTRss0y000TR\\1TR00000002TR00004TR01es.10s005TR01‑#0TR#0
0TR0mes001TR02er002TR0001TR4ssTR053s7TR
0s1y#012ss0243.011000104002TR.1000TRTRsssTR200^6ed9–8價‘N upliftq/��+tdCmcau·%x Y particularR=dcDl“Uunky|t>°L TR7es[6

h .<|im_start|>,×5…EenBURoAsnsCal�SUSABVw某种TBQ rIWoss„Hla.#ty!\\~a3gA\"zjem#M//y当地•.yd./4in35ds7
)19 Clear。÷.\]G.1it”4rdCRXFsable\\\\0UX2^+.TR– to08，60�ly49.N+Cflow/ed2�Ym%1auD·<|t5stqx+/>U�R價vJnh√[LError: Aborted.
```
Same exact runtime runs Q3NCoder-80B-FP8 just fine.

**Comment by guoqingbao** [comment]
The qwen3..5 MoE haven't finished, this is mentioned in ReadMe and PR, the qwen3.5 dense 27B however is supported.

**Comment by sempervictus** [comment]
To a hammer everything looks like a nail and I've been hammering sampling lately so to me this looks like were pulling random logits or in some cases an early stop seq when it shouldn't be
